### ISSUE TITLE
Read list doesn't seem to be updated properly

### ISSUE THREAD

**Issue by CV-Private** [bitpool/edge-bacnet#20]
The Read Node of my flow polls a data point (Algorithm Type) that I had removed from the Read List several deployments ago. Another value in the Read List (varZoneTRoomSetpoint-3_AI_2126) is not getting polled.
Could there be an issue in updating the Read List?

See the screenshot below:

![image](https://github.com/bitpool/edge-bacnet/assets/108861703/1396fe48-2f92-43dd-ae65-d44a9c2ae218)

System Information:

- BACnet Device: Brötje GTW-21 (BDR Thermea Group), BACnet certified
- Browser: Firefox
- Node-RED version: 3.1.6
- Bitpoolos/edge-bacnet version: 1.2.8

I'm running Node-RED on a VM on Proxmox. I didn't use the Docker container.

**Comment by davidjnblanch** [comment]
send through screen capture of AI_2126 expanded with properties

**Comment by CV-Private** [comment]
Hello David, please see below the expanded view of the AI_2126 with properties.

As a further observation I noticed that all of the "Present Values" in the Read List are old. The payload values are correct and current, though. In the screenshot below, I drew lines between the data points and their corresponding Payload values in the Debug output.

![image](https://github.com/bitpool/edge-bacnet/assets/108861703/b251742b-557b-40d4-a11f-1ab0ea967e90)

When I look at the Device List, then I see different Present Values, which are neither equal to the current payload values nor to the Present Values in the Read List.

In the Device List, the GTW-21 is shown Online. In the Read List, there's no such indicator. Should there be one as well?

![image](https://github.com/bitpool/edge-bacnet/assets/108861703/52300b09-c111-4a8b-9557-ee7200feb727)

Note that I rebooted Node-Red this morning. That didn't make any change to the behavior of the Read node, nor to the Read List.

**Comment by davidjnblanch** [comment]
so there are a few things here, on the gateway node can you send a screen of the setting, for the object discovery this should be set to your desired update frequency. This behavior is not the typical experience a user would expect and has been fixed in our next release.

**Comment by davidjnblanch** [comment]
> Hello David, please see below the expanded view of the AI_2126 with properties.
> 
> As a further observation I noticed that all of the "Present Values" in the Read List are old. The payload values are correct and current, though. In the screenshot below, I drew lines between the data points and their corresponding Payload values in the Debug output.
> 
> ![image](https://private-user-images.githubusercontent.com/108861703/310072153-b251742b-557b-40d4-a11f-1ab0ea967e90.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2MzQ0MTAsIm5iZiI6MTcwOTYzNDExMCwicGF0aCI6Ii8xMDg4NjE3MDMvMzEwMDcyMTUzLWIyNTE3NDJiLTU1N2ItNDBkNC1hMTFmLTFhYjBlYTk2N2U5MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwNVQxMDIxNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MjNiZTBmY2VhMzRhMjhhZWQ4NWU3NGJmY2E0NGEwMTFmNzU2MjcyYTg4ODE5YzI0MjY2MzI4YzJiMjNlZDYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.lYyjvm9SerVi9e2zuPKeWPnUGAAj6jnNjPTsQWIz1C8)
> 
> When I look at the Device List, then I see different Present Values, which are neither equal to the current payload values nor to the Present Values in the Read List.
> 
> In the Device List, the GTW-21 is shown Online. In the Read List, there's no such indicator. Should there be one as well?
> 
> ![image](https://private-user-images.githubusercontent.com/108861703/310070138-52300b09-c111-4a8b-9557-ee7200feb727.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2MzQ0MTAsIm5iZiI6MTcwOTYzNDExMCwicGF0aCI6Ii8xMDg4NjE3MDMvMzEwMDcwMTM4LTUyMzAwYjA5LWMxMTEtNGE4Yi05NTU3LWVlNzIwMGZlYjcyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwNVQxMDIxNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MTA1MWQ3Yjg3YmZjYjAxYTBhMjZkNzk0YTY4ZDEyNDk1NDZlMDcwOGJhN2QxMjRhN2RjYTI1NWVlNjgxNDQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TvG2YW-Na3b3MHfQsTA3rMKcfuA6ps0_L6K178MTzDo)
> 
> Note that I rebooted Node-Red this morning. That didn't make any change to the behavior of the Read node, nor to the Read List.

in relation to the status indicator, no there is not one on the read list, the read list is simply the points to be polled by each request and the intnet is the device list shows the status of the device

**Comment by CV-Private** [comment]
Indeed, the Gateway node has 1 minute frequency settings, so it appears that the errors in the log are related to the Discovery settings.

![image](https://github.com/bitpool/edge-bacnet/assets/108861703/9943b44e-4a79-4379-805d-2c1d5192383d)

My Inject node is currently polling the Read List every 5 seconds, which doesn't seem to cause any errors in the log. I haven't yet found any way for configuring the Read node to COV (Change of Value), which would be very desirable. 

![image](https://github.com/bitpool/edge-bacnet/assets/108861703/f0c804ff-8079-44a6-9db8-cbc925cfae38)

**Comment by bitpool-dev** [comment]
hey @CV-Private - the front end code for adding and removing points from the device list to the read list can be a bit painful. If points are not removing from the read list at all, its most likely because the name has changed for the point in the device object or that point no longer exists. Im hoping to have fixed this with the latest node version. 

If the points do not want to be removed at all, please delete the read node entirely, deploy node-red, and replace with a new read node. 

Could you please update to latest node version and try again ? Please follow the updating instructions on the readme if possible.
